test/unit/query/render-statictics.spec.js

```diff
@@ -0,0 +1,94 @@
+const should = require('should') // eslint-disable-line
```

Comment:
Notice how `should` is never used?  It's a weird thing about `should`.  You just require it in, and it adds itself to objects, etc.  But since you never actual use it, you will get a `declared but not used` linting error.

Reply:
Ahhh, okay. too bad you couldn't just require it in without declaring a variable. I've seen it like that for polyfills where you just have to `require('cross-fetch/polyfill');`